src/python/pants/backend/jvm/tasks/coverage/scoverage.py

```diff
@@ -150,6 +163,15 @@ def report(self, output_dir, execution_failed_exception=None):
     safe_mkdir(html_report_path, clean=True)
     safe_mkdir(xml_report_path, clean=True)
 
+    self._settings.log.info(f'output dir: {self._coverage_output_dir}')
+    if self._coverage_output_dir:
+      self._settings.log.debug(f'Scoverage output also written to: {self._coverage_output_dir}!')
+      mergetree(output_dir, self._coverage_output_dir)
+
+    import glob
```

Comment:
Should avoid inner-imports like this, unless you're doing it for a particular reason (in which case, a comment would be good).